index.html

```diff
@@ -180,27 +180,48 @@ <h2 class="title">Multi-Post Stories</h2>
       </section>
       <section class="contact-form">
         <div class="container">
-        <h2>Contact me</h2>
-        <p>
-          If you have an application you are interested in developing, a feature
-          that you need built or a project that needs coding. I’d love to help
-          with it
-        </p>
-        <form
-          action="https://formspree.io/f/mvolakkg"
-          method="POST"
-          class="form">
-          <input type="text" name="name" placeholder="Name" maxlength="30" />
-          <input type="email" name="email" id="" placeholder="Email" />
```

Comment:
Please it would be appropriate to provide a name for the input id, or better still remove it completely